recipe/meta.yaml

```diff
@@ -45,50 +52,51 @@ requirements:
     - wheel
   run:
     - python
-    - click >=7.0,<9.0
-    - cloudpickle <4.0
-    - databricks-cli >=0.8.7,<1
-    - entrypoints <1.0
-    - gitpython >=2.1.0,<4.0
-    - pyyaml >=5.1,<7.0
-    - protobuf >=3.12.0,<5.0
-    - pytz <2024
-    - requests >=2.17.3,<3.0
-    - packaging <24
+    - click >=7.0,<9
+    - cloudpickle <4
+    - entrypoints <1
+    - gitpython >=3.1.9,<4
     - importlib-metadata <8,>=3.7.0,!=4.7.0
-    - sqlparse >=0.4.0,<1.0
+    - packaging <25
+    - protobuf >=3.12.0,<6
+    - pytz <2025
+    - pyyaml >=5.1,<7
+    - requests >=2.17.3,<3
+    - sqlparse >=0.4.0,<1
 {% if mlflow_variant != "skinny" %}
-    - alembic <2,!=1.10.0
-    - docker-py >=4.0.0,<7.0
-    - flask <4.0
-    - numpy <2.0
-    - scipy <2.0
-    - pandas <3.0
-    - querystring_parser <2.0
-    - sqlalchemy >=1.4.0,<3
-    - gunicorn <22  # [not win]
-    - waitress <3  # [win]
-    - scikit-learn <2.0
-    - pyarrow <15,>=4.0.0
-    - markdown >=3.3,<4.0
+    - flask <4
     - jinja2 <4,>=2.11  # [not win]
     - jinja2 <4,>=3.0  # [win]
-    - matplotlib-base <4.0
-    # - prometheus_flask_exporter <1
+    - alembic <2,!=1.10.0
+    - docker-py >=4.0.0,<8
+    - graphene <4
+    - gunicorn <22  # [not win]
+    - markdown >=3.3,<4
+    - matplotlib-base <4
+    - numpy <2
+    - pandas <3
+    - pyarrow <16,>=4.0.0
+    - querystring_parser <2
+    - scikit-learn <2
+    - scipy <2
+    - sqlalchemy >=1.4.0,<3
+    - waitress <4  # [win]
 {% endif %}
   run_constrained:
     - mlflow{{ mlflow_other }} <0a0
     - mxnet !=1.8.0
     - fastai >=2.4.1
     - spacy >=3.3.0
-    - tensorflow >=2.8.0
+    - tensorflow >=2.10.0 # [not (osx and arm64)]
+    - tensorflow-macos >=2.10.0 # [osx and arm64]
```

Comment:
```suggestion
```
We don't have this package. Our tensorflow package is the `tensorflow-macos`.

Reply:
I thought that didn't matter if the package was listed under `run_constrained`?

Reply:
Usually yes, but in this case it's an ecosystem difference. `tensorflow-macos` is not and will never be valid in our ecosystem, not even in the future.

Reply:
To expand a little bit more, in our ecosystem, we made an explicit choice of having only one tensorflow package while the tensorflow team decided that they would have multiple tensorflow packages on PyPI. Their decision is valid in their context and our decision is valid in our context.

So in order to prevent confusion and future problems, it's better to remove `tensorflow-macos` from `run_constrained`.